tokio-trace/src/subscriber.rs

```diff
@@ -38,6 +38,8 @@ pub trait Subscriber {
         value: &dyn IntoValue,
     ) -> Result<(), AddValueError>;
 
+    fn add_follows_from(&self, span: &span::Id, follows: span::Id);
```

Comment:
Should this return an error if the span corresponding to that ID doesn't exist, like how `add_value` does?

Reply:
Yes, I think so. I can imagine two types of subscribers: 

- Subscribers that **do** care whether a span corresponds to that ID doesn't exist. An example might be a profiler like [pprof](https://github.com/google/pprof) or [hawktracer](https://github.com/amzn/hawktracer).
- Subscribers that **don't** care whether a span corresponds to that ID doesn't exist. An example might be a metrics emission system.

A metrics emission system could just ignore the error, but an implementer of a profiler might be frustrated if they don't get an error when calling this method. Introducing an error down the line would be a breaking error, so I'd opt for an error that can be ignored _now_.

Reply:
Yeah, that's reasonable. Good point.